npc/re/quests/quests_eclage.txt

```diff
@@ -2108,8 +2108,8 @@ ecl_fild01,97,315,4	script	Traveler#ep14_2	4_M_JOB_HUNTER,{
 		mes "[Traveler]";
 		mes "If you are thinking about selling it, you should just give up.";
 		next;
-		mes "I have no wishes to sell it no matter how much you beg.";
 		mes "[Traveler]";
+		mes "I have no wishes to sell it no matter how much you beg.";
```

Comment:
The liine are probably inverted in our aegis file too. The message should be like line 1953 https://github.com/keitenai/rathena/blob/75ec206d4a08784ebe2f1c62c143a80ccf46abcd/npc/re/quests/quests_eclage.txt#L1953